### DISCUSSION TITLE
Write HTTP Sender response to file

### DISCUSSION THREAD

**Discussion by murbanbsd** [nextgenhealthcare/connect#6195] [Q&A] [answered]
Hi, 
i try to write the response for GET to file but i am not able to read the response. Is there a solution? 

Thanks

**Comment by murbanbsd** [comment]
I solved it by myselft. 

in response transformers javascript: `
var myReponse = response.getMessage();`

**Reply by pacmano1** [reply, marked as answer]
The response to the GET is already "msg".   you don't have to call a method to retrieve it, it's already named.
